core/src/main/java/org/stellar/anchor/network/Rpc.java

```diff
@@ -0,0 +1,30 @@
+package org.stellar.anchor.network;
+
+import lombok.Getter;
+import org.stellar.anchor.config.AppConfig;
+import org.stellar.sdk.SorobanServer;
+import org.stellar.sdk.Transaction;
+import org.stellar.sdk.TransactionBuilderAccount;
+import org.stellar.sdk.responses.sorobanrpc.GetLatestLedgerResponse;
+import org.stellar.sdk.responses.sorobanrpc.SimulateTransactionResponse;
+
+@Getter
+public class Rpc {
```

Comment:
```suggestion
public class StellarRpc {
```

`Rpc` is too generic.